documentation/Extension_Point_Workflow.md

```diff
@@ -0,0 +1,108 @@
+# Workflow definitions
```

Comment:
Nitpick: All java code in this file is indented with 2 spaces.

Reply:
Punaisepoetser!